### PR TITLE
Adds support for WSL

### PR DESCRIPTION
**Pull request by campanalecp**
- adds is-wsl module as dependency
- adds additional 'wsl' string literal to PlatformTypes
- uses module to supplement os.platform() logic to account for possible
WSL host

Addresses #106

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,4632 @@
+{
```
Comment (RasPhilCo) [inline package-lock.json:1]: oclif uses yarn: can you delete this file and install the dep with yarn?

File: src/config.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import {CLIError, error, exit, warn} from '@oclif/errors'
+import WSL = require('is-wsl')
```
Comment (RasPhilCo) [inline src/config.ts:2]: though the linter doesn't catch this, can you move this below the imports and format as:

`const WSL = require('is-wsl')`

File: package-lock.json (hunk 1 above)
Comment (campanalecp) [reply package-lock.json]: done

File: src/config.ts (hunk 2 above)
Comment (campanalecp) [reply src/config.ts]: done; wasn't sure if you wanted it just after the imports or not so I moved it to a section above the primary class with similar kinds of utility functions

**Pull request merged**